src/rc.c

```diff
@@ -147,6 +149,12 @@ read_rc(const char *file)
 				} else {
 					options.netrc = 0;
 				}
+			} else if (strncmp("libsecret", vals[0], 9) == 0) {
```

Comment:
A thought: how about "keyring" or similar as the user-facing name for this config option as `libsecret` is more of an implementation detail and the user is more likely to be used to prompts from the likes of 'gnome-keyring' or 'kwallet' that actually implement the storage?

Reply:
(we'd better add to the man page too @t-brown!)

Reply:
I do like the word `keyring` better, however I am torn as we are using `libsecret`.  Does `kwallet` use `libsecret`? As it's GNOME vs KDE?

In hind sight, I should have done something like the following in the `rc` file:
```
pass = netrc | gpg | keyring 
```

Reply:
To be honest, I'm not sure how the potential backends are organised and could have been wrong to mention kwallet - I think the actual API name might be "Secrets Service". One could also perhaps generically call it a credentials cache.